test/app-tap/module_api.test.lua

```diff
@@ -2,8 +2,10 @@
 
 local ffi = require('ffi')
 local fio = require('fio')
+local msgpack = require('msgpack')
+local socket = require('socket')
 
-box.cfg{log = "tarantool.log"}
+box.cfg{listen = 3301, log = "tarantool.log"}
```

Comment:
This may break on CI, because the port may be busy. You should use a unix socket instead, as we do in other tests.

Reply:
Fixed: changed to unix socket.